appveyor.yml

```diff
@@ -19,12 +19,18 @@ environment:
     RUST_CONFIGURE_ARGS: --build=x86_64-pc-windows-msvc --enable-profiler
     SCRIPT: python x.py test
   - MSYS_BITS: 32
-    RUST_CONFIGURE_ARGS: --build=i686-pc-windows-msvc --target=i686-pc-windows-msvc
-    SCRIPT: python x.py test --host i686-pc-windows-msvc --target i686-pc-windows-msvc
+    RUST_CONFIGURE_ARGS: --build=i686-pc-windows-msvc
+    SCRIPT: python x.py test --exclude src/test/run-pass --exclude src/test/compile-fail
+  - MSYS_BITS: 32
+    RUST_CONFIGURE_ARGS: --build=i686-pc-windows-msvc
+    SCRIPT: python x.py test src/test/run-pass src/test/compile-fail
 
   # MSVC aux tests
   - MSYS_BITS: 64
-    RUST_CHECK_TARGET: check-aux
+    RUST_CHECK_TARGET: check-aux AUX_ARGS="--exclude src/tools/cargotest --exclude src/tools/cargo"
+    RUST_CONFIGURE_ARGS: --build=x86_64-pc-windows-msvc
+  - MSYS_BITS: 64
+    SCRIPT: python x.py test src/test/run-pass src/tools/cargotest src/tools/cargo
```

Comment:
Hm, this should probably be up above? The ordering here looks a little odd -- maybe I'm not entirely following though.

Reply:
If we are including `run-pass` here, the `x86_64-pc-windows-msvc` job (the 1st job) should have `--exclude run-pass`.

Reply:
Oh oops the `run-pass` was a mistake here, I'll remove that.